SonarQube.TeamBuild.PreProcessor/ProcessedArgs.cs

```diff
@@ -50,16 +51,21 @@ public ProcessedArgs(string key, string name, string version, bool installLoader
             {
                 throw new ArgumentNullException("globalFileProperties");
             }
+            if (scannerEnvProperties == null)
+            {
+                throw new ArgumentNullException("scannerEnvProperties");
```

Comment:
I remember that nameof(variable) made the build fail in some environment (I can't remember where, maybe in appveyor?)